css/obBase.css

```diff
@@ -134,6 +134,10 @@ form {
 #sideBar {
   position: absolute;
   right: 0;
+  top: 50px;
+  width:40px;
+  background-color: #252525;
+  height: 100%;
```

Comment:
Does height 100% work in this case?

If it doesn't, you can set bottom: 0, combined with top:50px that should make it fill the vertical space.

Position: fixed is better than absolute in this situation. That way it isn't affected by it's container.